packages/default/scss/tabstrip/_layout.scss

```diff
@@ -1,17 +1,28 @@
-$tabstrip-tab-padding-x: $nav-item-padding-x !default;
-$tabstrip-tab-padding-y: $nav-item-padding-y !default;
-$tabstrip-tab-gap: $nav-item-margin !default;
-
 @include exports("tabstrip/layout") {
 
+    .k-tabstrip-wrapper {
+        padding: $tabstrip-wrapper-padding-y $tabstrip-wrapper-padding-x;
+        border-width: $tabstrip-wrapper-border-width;
+        border-style: solid;
+        display: flex;
+        flex-direction: column;
+    }
+
     .k-tabstrip {
         border-width: 0;
+        border-color: transparent;
+        color: inherit;
         background-color: transparent;
         display: flex;
         flex-direction: column;
 
-        > .k-tabstrip-items {
-            border: 0 solid transparent;
+        .k-tabstrip-wrapper > & {
+            flex: 1 1 auto;
+        }
+
+        @at-root .k-tabstrip-items {
+            border-width: 0;
+            border-style: solid;
             border-color: inherit;
```

Comment:
Same question here? :)

Reply:
Here is a bit different. I am initially resetting the border to 0. Then in the tabstrip directional variants I am only setting the border-width that's relevant. That's applied to all themes with a variables, so that's why it's in the layout file.